src/vs/platform/terminal/common/terminal.ts

```diff
@@ -565,7 +565,7 @@ export interface IShellLaunchConfig {
 	 * until `Terminal.show` is called. The typical usage for this is when you need to run
 	 * something that may need interactivity but only want to tell the user about it when
 	 * interaction is needed. Note that the terminals will still be exposed to all extensions
-	 * as normal.
+	 * as normal. And the hidden terminals won't be restored when the workspace is next opened.
```

Comment:
I don't really like starting a sentence with "And"

Reply:
Oh same, I guess I didn't save that file when committing as it's meant to be the same

Reply:
Sorry about that. I don't know this at all.